content/spin/v2/kv-store-api-guide.md

```diff
@@ -57,7 +57,7 @@ fn handle_request(_req: Request) -> Result<impl IntoResponse> {
     store.set("mykey", "myvalue")?;
     let value = store.get("mykey")?;
     let response = value.unwrap_or_else(|| "not found".into());
-    Ok(http::Response::builder().status(200).body(response)?)
+    Ok(Response::builder().status(200).body(response)?)
```

Comment:
This is not correct. The sdk `Response` builder does not return a `Result` from the `body` method like the `http` crate's `ResponseBuilder` does. 

This should be the following I believe (but I didn't run this through a compiler so you should probably do that to make sure this works):

```rust
Ok(Response::builder().status(200).body(response).build())
```

Reply:
Thanks,
I will update these and test theem also.
Great spotting; thank you!!!